.pyup.yml

```diff
@@ -0,0 +1,21 @@
+# PyUp config
+# https://pyup.io/docs/bot/config/
+# Check dependencies in _only_ requirements-app, and open PRs with PyUp prefix.
+
+search: False
+schedule: "every week on sunday"
+
+requirements:
+
+  - requirements-app.txt:
```

Comment:
We don't have a requirements-app for the AWS repo, just requirements.txt.

Reply:
🤦‍♂ good point